GameServer/Model/AgarioGame.cs

```diff
@@ -53,7 +52,8 @@ private void CheckCollisions()
                         secondEntity))
                     {
                         if (firstEntity.Radius > secondEntity.Radius &&
-                            firstEntity.EntityType == EntityType.Player)
+                            firstEntity.EntityType == EntityType.Player &&
+                            !firstEntity.IsDead && !secondEntity.IsDead)
                         {
                             Player player = (Player)firstEntity;
                             player.Kill(secondEntity);
```

Comment:
```suggestion
                            player.Kill(secondEntity);
                            Board.RemoveEntityFromBoard(secondEntity);
```
This will fix bug with view

Reply:
True, ty